firecloud/fccore.py

```diff
@@ -162,6 +165,14 @@ def config_parse(config=None, *files):
     if os.path.isfile(config.credentials):
         os.environ[environment_vars.CREDENTIALS] = config.credentials
 
+    # if config override file options with passed options
+    for key, value in config.items():
+        config[key] = value
```

Comment:
I'm not sure I understand what's supposed to be happening here, config is simply being set to itself, this looks equivalent to:

```python
for key in config:
    config[key] = config[key]
```
Overrides should probably happen in the loop at line 153, otherwise it looks like they've potentially been wiped out at this point in the code.